### PR TITLE
:art: added AdaptorsDependencies module

### PR DESCRIPTION
**Pull request by cygnus-x-1**
### Description of the Change

Added AdaptorsDependencies module to support any dependencies (additional to those in Constellation Core) required by adaptors.

Added AdaptorsMapView module to support map plugins and added several map provider implementations, including ArcGIS Online, Bing Maps and Open Street Maps.

### Alternate Designs

N/A

### Why Should This Be In Core?

This makes it easier for contributors to add dependencies, and provides support for web maps in the Map View.

### Benefits

This makes it easier for contributors to add dependencies, and provides support for web maps in the Map View.

### Possible Drawbacks

N/A

### Verification Process

Everything builds, and all map providers render efficiently across all tiles.

### Applicable Issues

Some map providers (Google Maps) have limitations which prevent extended usage, resulting in the map returning HTTP error codes upon tile requests. It might be worth disabling those providers at some point if a resolution cannot be found.

### CODE REVIEW

File: AdaptorsDependencies/nbproject/project.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+javac.source=1.7
```
Comment (arcturus2) [inline AdaptorsDependencies/nbproject/project.properties]: 1.8?

File: AdaptorsMapView/nbproject/project.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+javac.source=1.7
```
Comment (arcturus2) [inline AdaptorsMapView/nbproject/project.properties]: 1.8?

File: AdaptorsDependencies/nbproject/project.properties (hunk 1 above)
Comment (cygnus-x-1) [reply AdaptorsDependencies/nbproject/project.properties]: Fixed.

File: AdaptorsMapView/nbproject/project.properties (hunk 2 above)
Comment (cygnus-x-1) [reply AdaptorsMapView/nbproject/project.properties]: Fixed.

**Pull request merged**